src/Datasource/RepositoryInterface.php

```diff
@@ -17,6 +17,8 @@
 /**
  * Describes the methods that any class representing a data storage should
  * comply with.
+ *
+ * @method int deleteEach($conditions, $options = []);
```

Comment:
This suggests that other implementations of `RepositoryInterface` have `deleteEach()` method, but they probably don't.